q2_types/distance_matrix/tests/test_type.py

```diff
@@ -0,0 +1,34 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import unittest
+
+from q2_types.plugin_setup import plugin
+from q2_types.distance_matrix import (DistanceMatrix,
+                                      DistanceMatrixDirectoryFormat)
+from q2_types.testing import TestPluginBase
+
+
+class TestTypes(TestPluginBase):
+    def test_distance_matrix_semantic_type_registration(self):
+        self.assertEqual(plugin.types['DistanceMatrix'].semantic_type,
+                         DistanceMatrix)
```

Comment:
This should be a method/helper I think:

```python
self.assertEqualSemanticType('DistanceMatrix', DistanceMatrix)
```

Not sold on that name, but we shouldn't have to use the plugin object in these tests ideally.